DXRando/DeusEx/Classes/DXRMissionEndgame.uc

```diff
@@ -270,6 +272,7 @@ function Timer()
 
         if (endgameTimer > 75) {
             FinishCinematic();
+            SpawnDXRando(p);
```

Comment:
Calling SpawnDXRando here won't do anything, because by the time DXRMissionEndgame has run, that implies there was already a DXRando to have replaced the mission script.  SpawnDXRando needs to be called in PostLogin of DXRandoGameInfo (probably - I know there were timing concerns with the AlarmLight kludge, so PostLogin might still need a delay?)